frameworks/cassandra/alerts/README.md

```diff
@@ -0,0 +1,27 @@
+# Cassandra alerts for DC/OS
+
+In order to setup alerts on your cluster, you must install the following two packages:
+- `dcos-monitoring`
+- `cassandra`
+
+1. Save the following to an `options.json` file:
+```
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "url": "https://github.com/mesosphere/dcos-commons",
+      "path": "/frameworks/cassandra/alerts/",
+      "reference_name": "refs/heads/cassandra-dashboards-and-alerts"
```

Comment:
this is fine for now but we'll have to change when we merge to the dcos/alerts repo

Reply:
@samvantran noted !!